docs/api/session.md

```diff
@@ -523,3 +523,24 @@ The `listener` will be called with `listener(details)` when an error occurs.
   * `timestamp` Double
   * `fromCache` Boolean
   * `error` String - The error description.
+
+#### `ses.protocol`
+
+Returns an instance of [protocol](protocol.md) module for this session.
+
+```javascript
+const electron = require('electron');
+const app = electron.app;
+const path = require('path');
+const protocol = session.fromPartition(partitionName).protocol;
```

Comment:
Creating session before the ready event is not recommended, for now it is causing crash on exit.